owners/index.js

```diff
@@ -43,11 +43,19 @@ module.exports = app => {
    */
   async function runOwnersCheck(github, pr) {
     const ownersCheck = new OwnersCheck(github, pr);
-    const fileOwners = await Owner.getOwners(github, pr.number);
-    const approvers = await ownersCheck.getApprovers();
+    let checkRunId;
+    let latestCheckRun;
 
-    const checkRunId = await github.getCheckRunId(pr.headSha);
-    const latestCheckRun = ownersCheck.buildCheckRun(fileOwners, approvers);
+    try {
+      const approvers = await ownersCheck.getApprovers();
+      checkRunId = await github.getCheckRunId(pr.headSha);
+      const fileOwners = await Owner.getOwners(github, pr.number);
```

Comment:
This call to `getOwners` is what throws the error in some cases. It will be replaced in an upcoming PR, so identifying and patching the underlying (and rare) bug is not worth the time and effort.

Reply:
You can start the `try` clause just before this line then, unless you want all of the above statements to create the fallback check-run if they fail for any reason

Reply:
I want it to always create a check-run, even if there's some issue getting something from GitHub or elsewhere.